README.md

```diff
@@ -38,6 +38,7 @@ This translates into the following steps:
 3. Go to the **image** directory and run `terraform init && terraform apply`
 4. Go to the **libvirt-network** directory and run: `terraform init && terraform apply`
 5. Go to the **etcd** directory and run `terraform init && terraform apply`
+    1. If you get `Permission denied` errors, it's probably because SELinux is not enabled on your system and QEMU enforces it. To fix that, its security driver can be disabled (not a good security practice, but acceptable for a local environment). To do so, add `security_driver = "none"` to `/etc/libvirt/qemu.conf` and restart libvirt (`sudo systemctl restart libvirtd`).
```

Comment:
That's not the way I'd go about it. We can discuss it tomorrow.

On ubuntu with apparmor, there is a file you can edit to add permissions.

Reply:
Ok I will test that and let you know.

Reply:
Done.